aten/src/ATen/native/native_functions.yaml

```diff
@@ -1841,11 +1850,18 @@
   variants: function
 
 
-- func: sparse_raw_resize_(Tensor self, IntList size, int64_t sparseDims, int64_t denseDims) -> Tensor
+# Remove this function when USE_TH_SIZE_ZERO_DIM is enabled by default
+- func: sparse_raw_resize_legacy_(Tensor self, IntList size, int64_t sparseDims, int64_t denseDims) -> Tensor
+  variants: method
+  dispatch:
+    SparseCPU: raw_resize_sparse_legacy_
```

Comment:
I didn't reorder the words in the function name actually - it's the same order in the original function (I can fix them to have the same order if needed). I added a `legacy_` suffix to it so that it's more clear in the call sites that this is a legacy function.